cli/initiator/reshare.go

```diff
@@ -86,26 +87,27 @@ var GenerateReshareMsg = &cobra.Command{
 				ethNetwork,
 				cli_utils.WithdrawAddress[:],
 				cli_utils.OwnerAddress,
-				nonce, 
+				nonce,
 				cli_utils.Amount,
 				signedProofs[i],
 			)
 			if err != nil {
-				logger.Fatal("😥 Failed to construct reshare message: ", zap.Error(err))
```

Comment:
Maybe keep this log as it was because this error is constructing the actual message